CHANGELOG

```diff
@@ -1,4 +1,5 @@
 0.54.3
+ - ref: replace lru_cache on instance methods with cache attributes
```

Comment:
I think this is a "fix". Please reference the issue number.

Reply:
Sounds good!